crates/notary/server/fixture/auth/whitelist_copied.csv

```diff
@@ -0,0 +1,4 @@
+"Name","ApiKey","CreatedAt"
```

Comment:
You accidentally checked in this I believe

Reply:
I was running some unit tests, I believe it was automatically generated. removing it now.